core/server/models/member.js

```diff
@@ -0,0 +1,65 @@
+const ghostBookshelf = require('./base');
+
+ghostBookshelf.model('MemberPassword', ghostBookshelf.Model.extend({
```

Comment:
Seems reasonable - what do you think?

Reply:
Yeah i think so as well, specially with the current implementation. Seems overkill to extract out in a file for this 😄